Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -69,14 +69,22 @@
         {
             "name": "WebAppKind",
             "type": "pickList",
-            "label": "Web app kind",
+            "label": "App Type",
             "defaultValue": "app",
             "required": false,
```

Comment:
should we set the required property to true? (since the default value will be populated for old RD, if there is no app type)